lib/l10n/app_it.arb

```diff
@@ -0,0 +1,260 @@
+  "loading": "Caricamento...",
+  "artist": "Artista",
+  "blacklisted": "Blacklisted",
+  "following": "Following",
+  "follow": "Follow",
+  "artist_url_copied": "URL dell'Artista copiato nella clipboard",
+  "added_to_queue": "Aggiunto {tracks} alla coda",
+  "filter_albums": "Filtra album...",
+  "synced": "Sincronizzato",
+  "plain": "Plain",
+  "shuffle": "Shuffle",
+  "search_tracks": "Cerca brani...",
+  "released": "Rilasciato",
+  "error": "Errore {error}",
+  "title": "Titolo",
+  "time": "Time",
+  "more_actions": "Altre azioni",
+  "download_count": "Download ({count})",
+  "add_count_to_playlist": "Aggiungi ({count}) alla Playlist",
+  "add_count_to_queue": "Aggiungi ({count}) alla Coda",
+  "play_count_next": "Riproduci ({count}) seguenti",
+  "album": "Album",
+  "copied_to_clipboard": "Copiato {data} nella clipboard",
+  "add_to_following_playlists": "Aggiungi {track} alle Playlist successive",
+  "add": "Aggiungi",
+  "added_track_to_queue": "Aggiunta {track} alla coda",
+  "add_to_queue": "Aggiungi alla coda",
+  "track_will_play_next": "{track} verrà riprodotta dopo",
+  "play_next": "Riproduci seccessivo",
+  "removed_track_from_queue": "Rimossa {track} dalla coda",
+  "remove_from_queue": "Rimuovi dalla coda",
+  "remove_from_favorites": "Rimuovi dai preferiti",
+  "save_as_favorite": "Salva come preferito",
+  "add_to_playlist": "Aggiungi alla playlist",
+  "remove_from_playlist": "Rimuovi dalla playlist",
+  "add_to_blacklist": "Aggiungi alla blacklist",
+  "remove_from_blacklist": "Rimuovi dalla blacklist",
+  "share": "Condividi",
+  "mini_player": "Mini Player",
+  "slide_to_seek": "Slide to seek forward or backward",
+  "shuffle_playlist": "Shuffle playlist",
+  "unshuffle_playlist": "Unshuffle playlist",
+  "previous_track": "Traccia Precedente",
+  "next_track": "Traccia Successiva",
+  "pause_playback": "Pausa Playback",
+  "resume_playback": "Riprendi Playback",
+  "loop_track": "Brano in Loop",
+  "repeat_playlist": "Ripeti playlist",
+  "queue": "Coda",
+  "alternative_track_sources": "Sorgenti dei brani Alternative",
+  "download_track": "Scarica Brano",
+  "tracks_in_queue": "{tracks} brani in coda",
+  "clear_all": "Pulisci Tutto",
+  "show_hide_ui_on_hover": "Mostra/Nascondi UI al passaggio del mouse",
+  "always_on_top": "Sempre in primo piano",
+  "exit_mini_player": "Esci da Mini player",
+  "download_location": "Cartella Download",
+  "account": "Profilo",
+  "login_with_spotify": "Entra con il tuo profilo Spotify",
+  "connect_with_spotify": "Connettiti con Spotify",
+  "logout": "Esci",
+  "logout_of_this_account": "Esci da questo profilo",
+  "language_region": "Lingua & Paese",
+  "language": "Lingua",
+  "system_default": "Default di Sistema",
+  "market_place_region": "Paese del Marketplace",
+  "recommendation_country": "Recommendation Country",
+  "appearance": "Aspetto",
+  "layout_mode": "Layout Mode",
+  "override_layout_settings": "Override responsive layout mode settings",
```

Comment:
@OpenCode should we translate it ?

Reply:
Yeah. Have you any idea?

Reply:
What about "Sovrascrivi le impostazioni del layout responsivo" ?

Reply:
It can be good